components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/util/OAuth2UtilTest.java

```diff
@@ -1027,7 +1029,7 @@ public void testGetOAuth2TokenEPUrl(String configUrl, String serverUrl, String o
                 {true, "https://localhost:9443/testUrl", "https://localhost:9443/testUrl", "testDomain",
                         "https://localhost:9443/t/testDomain/oauth2/jwks"},
                 {true, "", "https://localhost:9443/testUrl", "",
-                        "https://localhost:9443/t/carbon.super/oauth2/jwks"},
+                        "https://localhost:9443/oauth2/jwks"},
```

Comment:
@ChamathNS Is this the expected behaviour of this test?

Reply:
Yes